tests/run/xml-interpolation/XmlQuote_1.scala

```diff
@@ -0,0 +1,61 @@
+import scala.quoted._
+import scala.tasty.Tasty
+
+import scala.language.implicitConversions
+
+case class Xml(parts: String, args: List[Any])
+
+// Ideally should be an implicit class but the implicit conversion
+// has to be a rewrite method
```

Comment:
Good point, I think this is the best solution. Otherwise, we would need something like `implicit rewrite class XmlQuote(...)`. Maybe with proper extension methods this will not be an issue.